code/modules/research/designs_vr.dm

```diff
@@ -87,7 +87,7 @@
 /datum/design/item/translocator
 	name = "Personal translocator"
 	id = "translocator"
-	req_tech = list(TECH_MAGNET = 5, TECH_BLUESPACE = 5, TECH_ILLEGAL = 7)
+	req_tech = list(TECH_MAGNET = 5, TECH_BLUESPACE = 5, TECH_ILLEGAL = 6)
```

Comment:
I actually kind of wish some things _were_ unobtainable high, forcing R&D to find materials outside of the station.